### PR TITLE
Jake

### PR DESCRIPTION
**Pull request by JakeDish**

### CODE REVIEW

File: assets/js/quiz.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var wrongA
+var rightA
+var shuffled
+var quizOver = false
+
+
+function displayQuestion() {
+  if (quizOver = false) {
```
Comment (ericnguyen23) [inline assets/js/quiz.js:8]: You need to do a strict comparison with 3 === 
`quizOver = false`

File: quiz.html [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
-        <button class="btn">Answer 2</button>
-        <button class="btn">Answer 3</button>
-        <button class="btn">Answer 4</button>
+        <button class="btn" id="a1">Answer 1</button>
+        <button class="btn" id="a2">Answer 2</button>
+        <button class="btn" id="a3">Answer 3</button>
+        <button class="btn" id="a4">Answer 4</button>
       </div>
     </div>
     <div class="controls">
```
Comment (ericnguyen23) [inline quiz.html:23]: On the version I have, it was linked to script.js: `<script src="./assets/js/script.js"></script>`
Just verify that your's is linking to quiz.js: `<script src="./assets/js/quiz.js"></script>`

**Review by ericnguyen23 — CHANGES_REQUESTED** [review]
I just ad a couple of comments. Once you resolve, we can approve and merge.

File: assets/js/quiz.js (hunk 1 above)
Comment (JakeDish) [reply assets/js/quiz.js]: updated to ===

File: quiz.html (hunk 2 above)
Comment (JakeDish) [reply quiz.html:23]: I created a new js file for this since I wasn't sure if anyone else was using the script.js file. Updated the script link

**Review by ericnguyen23 — APPROVED** [review]

**Pull request merged**